### ISSUE TITLE
Design: Accessibility Tab

### ISSUE THREAD

**Issue by Jesse-Box** [getsentry/sentry#57550]
### Problem
- Users don't have a convenient way to understand when certain accessibility rules have been violated within their application

### Acceptance Criteria
TBD

**Comment by JoshFerge** [comment]
closing as accessibility project has been postponed for a while.
